apps/meteor/ee/client/omnichannel/slaPolicies/SlaEdit.tsx

```diff
@@ -53,16 +54,16 @@ function SlaEdit({ data, isNew, slaId, reload, ...props }: SlaEditProps): ReactE
 	const handleSave = useMutableCallback(async () => {
 		const { name, description, dueTimeInMinutes } = getValues();
 
-		if (!isValid) {
+		if (!isValid || !name || dueTimeInMinutes === undefined) {
```

Comment:
Shouldn't this validations be handled in the form instead of the onSave? @MartinSchoeler maybe you can help here :eyes:

Reply:
I believe there's already a validation over there however I did this just to avoid typescript errors, because, below typescript was still resolving it to be undefined.